models/posts.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require("mongoose");
+
+let Posts = new mongoose.Schema({
+  title: String,
+  description: String,
+  comments: [{ type: mongoose.Schema.Types.ObjectId, ref: "Comment" }],
+  userId: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
+  date: { type: Date, default: Date.now },
+});
+
+module.exports = mongoose.model("Posts", Posts);
```

Comment:
`Post` instead of `Posts`.

Also this file name should be `post.js`.

Reply:
updated